MCLEN/MCLEN-IOC-01App/Db/mclen_motor_extra.db

```diff
@@ -2,4 +2,5 @@ record(motor, "$(P)$(M)") {
   field(PCOF, "$(PCOF)")
   field(HVEL, "$(HVEL)")
   field(RDBD, "$(RDBD)")
+  field(ASG,  "WASL0")
```

Comment:
everything including the SPs and home/stop buttons are disabled for me - do I need some sort of access control file?

Reply:
I can still change OFF and BDST - probably don't want to do at least the first one?

Reply:
We do set BSDT on some Galil systems, in fact SANS2D tank might be one of them, but probably won't set it on a mclennan as it has its own backup steps controller parameter. My concern with protecting OFF/SET is don't reflectometer scientists use it - home to neutrons and then define that as zero?

Reply:
While looking at this is did however make me realise we need to protect the status update button too https://github.com/ISISComputingGroup/IBEX/issues/7837